pkg/segment/results/segresults/segresults.go

```diff
@@ -88,11 +88,11 @@ type SearchResults struct {
 	runningEvalStats       map[string]interface{}
 	segStatsResults        *segStatsResults
 	convertedBuckets       map[string]*structs.AggregationResult
-	allSSTS                map[uint16]map[string]*structs.SegStats // maps segKeyEnc to a map of segstats
+	allSSTS                map[uint32]map[string]*structs.SegStats // maps segKeyEnc to a map of segstats
 	AllErrors              []error
-	SegKeyToEnc            map[string]uint16
-	SegEncToKey            map[uint16]string
-	NextSegKeyEnc          uint16
+	SegKeyToEnc            map[string]uint32
+	SegEncToKey            map[uint32]string
```

Comment:
FYI there's a `utils.TwoWayMap` type to handle cases where we want both forward and reverse maps. (I don't think you should change these to that in this PR, just wanted to let you know)